NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -3649,6 +3671,17 @@ int HTauTauNtuplizer::FillJet(const edm::View<pat::Jet> *jets, const edm::Event&
 
   return nJets;
 }
+void HTauTauNtuplizer::FillBoostedTaus(const edm::View<pat::Tau>* boostedTaus, const edm::Event&)
+{
+  //TODO
+    for(edm::View<pat::Tau>::const_iterator itau = boostedTaus->begin(); itau!=boostedTaus->end();++itau)
+    {
+      _boostedTaus_px.push_back( (float) itau->px()     );
+      _boostedTaus_py.push_back( (float) itau->py()     );
+      _boostedTaus_pz.push_back( (float) itau->pz()     );
+      _boostedTaus_e.push_back(  (float) itau->energy() );
+    }
```

Comment:
Just a few ideas on what branches could be useful:
- trigger information if we plan to add some triggers using boosted taus (not even sure if they exist!)
- deepTau scores or ele/mu isolation values
- possible Energy Scale shifts
- gen matching (real tau or e/mu faking tau)
Also, most variables needed for the even selection is defined in [OfflineProducerHelper](https://github.com/LLRCMS/KLUBAnalysis/blob/VBF_legacy/src/OfflineProducerHelper.cc) in KLUB, it's worth having a look at that as well!

Reply:
I did not find any info on boosted tau triggers, but if there are some it'd definitely be interesting. The isolation and ID variables are now included. I'll still need to add the TES and gen variables.

Reply:
Hi Louis! In my opinion we can keep only the deepTauv2p1 iso variables... no need for MVA, againstEle, againsMuon